homeworks/valitothattila/4_exceptions_logging/to_do.py

```diff
@@ -0,0 +1,103 @@
+import os
+import logging
+
+print(os.getcwd())
+
+file_path = "python_hu/homeworks/valitothattila/4_exceptions_logging/to_do.txt"
+
+
+# ------------------------ Logolás beállítása -------------------------
+file_handler = logging.FileHandler("python_hu/homeworks/valitothattila/4_exceptions_logging/app.log")
+stream_handler = logging.StreamHandler()
+
+formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+
+file_handler.setFormatter(formatter)
+stream_handler.setFormatter(formatter)
+
+file_handler.setLevel(logging.DEBUG)
+stream_handler.setLevel(logging.DEBUG)
+
+logger = logging.getLogger()
+logger.addHandler(file_handler)
+logger.addHandler(stream_handler)
+logger.setLevel(logging.DEBUG)
+
+# ------------------------ Függvények -------------------------
+
+# ------------------------ Menü = 1 -> task_list elem hozzáadása  -------------------------
+def add_to_tasklist(add_element):
+    with open(file_path, "a") as file:
```

Comment:
Ide is kéne egy exception kezelés, legalább egy general (`try except Exception as e:`), csak azért, hogy ha valami hiba történik a file-ba való íráskor, akkor ne álljon le a programod, csak logoljuk ki a hibát.